pkgs/by-name/sh/shira/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication {
+  pname = "shira";
+  version = "unstable-2025-08-31";
```

```suggestion
  version = "0-unstable-2025-08-31";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning